### ISSUE TITLE
PSF layer install error

### ISSUE THREAD

**Issue by sunnyHelen** [shirgur/UnsupervisedDepthFromFocus#1]
Thank you for sharing your work. I'd like to try the code, but the first step installing is with something wrong. Could you please help me?
![image](https://user-images.githubusercontent.com/28101548/62762012-7bd20680-bacb-11e9-8bfe-b888c825a3de.png)

**Comment by everythoughthelps** [comment]
i met this problem too,have you solve it?

**Comment by fuy34** [comment]
I solved it by replacing the original path in the ``` setup.py```  with absolute path.

**Comment by Crispig** [comment]
> 
> 
> I solved it by replacing the original path in the ` setup.py` with absolute path.

You are right! replace "gauss_psf_kernel.cu" with "gauss_psf_cuda_kernel.cu" in _setup.py_

**Comment by xuejiancai** [comment]
> > I solved it by replacing the original path in the ` setup.py` with absolute path.
> 
> You are right! replace "gauss_psf_kernel.cu" with "gauss_psf_cuda_kernel.cu" in _setup.py_

I do the same with you,but it got error:
g++: error: /home/xxx/torch_project/UnsupervisedDepthFromFocus-master/UnsupervisedDepthFromFocus-master/ext/build/temp.linux-x86_64-3.8/gauss_psf_cuda.o: 没有那个文件或目录
g++: error: /home/xxx/torch_project/UnsupervisedDepthFromFocus-master/UnsupervisedDepthFromFocus-master/ext/build/temp.linux-x86_64-3.8/gauss_psf_cuda_kernel.o: 没有那个文件或目录
could you help me? @Crispig thx!

**Comment by Crispig** [comment]
I forget the details of this project, but I remember that there is a difference between the call to some files in the setup file and the actual file names, so you can pay attention to that. @xuejiancai

**Comment by xuejiancai** [comment]
> I forget the details of this project, but I remember that there is a difference between the call to some files in the setup file and the actual file names, so you can pay attention to that. @xuejiancai

ok,thx!

**Comment by Moushumi9medhi** [comment]
In case this still helps anyone, I solved it by renaming the "gauss_psf_kernel.cu" with "gauss_psf_cuda_kernel.cu" in setup.py and copying all the files inside the ext/ directory to the main folder. Then I had run the setup file inside the main folder.
